doc/changelog/08-cli-tools/14024-coqdep-errors.rst

```diff
@@ -0,0 +1,8 @@
+- **Changed:**
+  Behavior of coqdep for unknown options and non-existing files.
+  Files that don't exist or are unreadable will cause coqdep to
+  fail with exit status 1. For unknown options, coqdep will emit
```

Comment:
Maybe add "whereas before ``coqdep`` did ignore them previously" ?

Not sure if coqdep has to be written using some Sphinx concrete syntax or link.

Reply:
```suggestion
  fail with exit status 1. For unknown options, ``coqdep`` will emit
```

There's currently no hyperlink target defined for coqdep (in 2 places).  It would be good to enclose in double backticks to make it bold, though.

Reply:
it is now inside backticks